# HG changeset patch
# User Rik <rik@octave.org>
# Date 1452894253 28800
#      Fri Jan 15 13:44:13 2016 -0800
# Node ID 706dcaf7bf5fbf548bc8d5220e99244cf57c365a
# Parent  49852ff04747f9217dfa02ca7cc134635e15adee
maint: Remove unnecessary retval declaration from ov-fcn-handle.cc.

 *ov-fcn-handle.cc: Remove unnecessary retval declaration.

diff --git a/libinterp/octave-value/ov-fcn-handle.cc b/libinterp/octave-value/ov-fcn-handle.cc
--- a/libinterp/octave-value/ov-fcn-handle.cc
+++ b/libinterp/octave-value/ov-fcn-handle.cc
@@ -1676,18 +1676,16 @@ In addition, some function types may ret
 fields.\n\
 \n\
 @strong{Warning:} @code{functions} is provided for debugging purposes only.\n\
 It's behavior may change in the future and programs should not depend on a\n\
 particular output.\n\
 \n\
 @end deftypefn")
 {
-  octave_value retval;
-
   if (args.length () != 1)
     print_usage ();
 
   octave_fcn_handle *fh = args(0).fcn_handle_value ("functions: FCN_HANDLE argument must be a function handle object");
 
   octave_function *fcn = fh ? fh->function_value () : 0;
 
   if (! fcn)
